antora-ui-camel/src/css/frontpage.css

```diff
@@ -97,7 +97,6 @@ div.frontpage.news {
   text-align: center;
   margin-left: 3rem;
   background-color: var(--color-smoke-50);
-  border-radius: 1rem 1rem 0 0;
```

Comment:
This border radius is for for the "what's new" title div on desktop view. I think we can keep this..